footprints/main/utils.py

```diff
@@ -124,6 +124,10 @@ def get_geoname_by_id(self, gid):
                'username={}&type=json&geonameId={}').format(
                    settings.GEONAMES_KEY, gid)
         results = requests.get(url)
+
+        if results.status_code == 404:
+            raise ValueError(f"Geoname ID {gid} not found.")
```

Comment:
Just a minor note: we can use a single-quote for this f-string here to adhere with the rest of our python standards.

https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals